File: web-shop/src/main/webapp/js/service/seckill_goodsService.js
//服务层
app.service('seckill_goodsService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../seckill_goods/findAll.do');
	}

	//查询实体
	this.findOne=function(id){
		return $http.get('../seckill_goods/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../seckill_goods/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../seckill_goods/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../seckill_goods/delete.do?ids='+ids);
	}
    //提交审核
    this.updateStat=function(ids){
        return $http.get('../seckill_goods/updateStat.do?ids='+ids);
    }
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../seckill_goods/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: interface/src/main/java/cn/itcast/core/service/Seckill_goodsService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.seckill.SeckillGoods;
import org.springframework.stereotype.Repository;

@Repository
public interface Seckill_goodsService {
    //添加商品
    void add(SeckillGoods seckillGood);

    //分页加条件查询
    PageResult search(Integer page, Integer rows, SeckillGoods seckillGoods);

    //点修改回显数据
    SeckillGoods findOne(Long id);

    //修改
    void update(SeckillGoods seckillGoods);

    //删除(逻辑删除,并不是真的在数据库中删除,只是在页面上不显示)
    void delete(Long[] ids);

    //批量修改状态
    void updateStat(Long[] ids);

}
